pimpmycause/pimpmycause/urls.py

```diff
@@ -17,16 +17,16 @@
 from django.contrib import admin
 
 from core.views import HomepageView, ContactView, AboutView, MeetTheTeam, OurPartners, Partnerships, BecomeAPartner, WhoWeAre, HowItWorks
-from profiles.views import LoginView, RegistrationView, RegistrationComplete, ActivationComplete, profile_update
+from profiles.views import RegistrationView, RegistrationComplete, ActivationComplete, profile_update, logout_view
 from profiles.forms import PimpUserRegistrationForm
 
 urlpatterns = [
     url(r'^admin/', admin.site.urls),
     url(r'^$', HomepageView.as_view(), name="index"),
-    url(r'^login/', LoginView.as_view(), name="login"),
+    url(r'^logout/', logout_view, name="logout_view"),
```

Comment:
The logout view is different from others it is not a Class Based view but it is actually a function based view like profile_page, I am following Pep8 convention as advised by Django docs: https://www.python.org/dev/peps/pep-0008/#function-names and https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/